structs.go

```diff
@@ -794,6 +794,9 @@ type Member struct {
 	// Whether the member is muted at a guild level.
 	Mute bool `json:"mute"`
 
+	// The avatar of the member for the guild, if any.
+	Avatar string `json:"avatar"`
```

Comment:
The doc-comment might be a little confusing. Since this is a hash, not the avatar url.

Reply:
Still no change in the comment. Maybe you forgot to push?